startserver.ps1

```diff
@@ -0,0 +1,17 @@
+#!/bin/env pwsh
+$pythonversions = @('pypy3','pypy','py','python3','python')
+$minimumVersion = '3.7.9'
+
+foreach ($command in $pythonversions) {
+        if(Get-Command -ErrorAction SilentlyContinue $command){
+            $currentver=Invoke-Expression -Command "$command -c 'import platform;print(platform.python_version())'"
+            if ($currentver -ge $minimumVersion) {
+                Write-Output "Using Python version $currentver ($command)"
+                Invoke-Expression "$command pymine"
+                exit 0
+            }
+        }
+}
+
+Write-Output "Couldn't find suitable Python version. Please use your package manager to install Python 3.7.9 or later"
```

Comment:
Perhaps instead of telling them to install Python via a package manager (Is that even a thing?) Tell them to download it from python.org, and perhaps link them to the downloads page (`python.org/downloads`)

Reply:
https://chocolatey.org/ and the Microsoft store could both be considered package managers,
But I see what you mean

Reply:
The Microsoft store is a horrible place to install Python, and not every user has chocolatey. At the very least, you should change "Please use your package manager...." to "Please install Python 3.7.9 or later", and let the user decide how they want to install it. The reason for including a download link for Python in the message would be for non-developers trying to use PyMine.

Reply:
:+1: